wiki/fr/fr-Installation-for-Windows.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: wiki
+title: "Installation sous Windows"
+lang: "fr"
+permalink: "/wiki/Installation-for-Windows"
+---
+
+# Installation sous Windows
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+1. **Téléchargez et installez un pilote ASIO**. Il est recommandé d'utiliser une carte/interface son avec un pilote ASIO natif. Si vous n'en avez pas (en particulier pour les cartes son internes), installez ce [pilote ASIO gratuit (ASIO4All)](http://www.asio4all.org){: target="_blank" rel="noopener noreferrer" title="ASIO4ALL - Universal ASIO Driver For WDM Audio"} avant d'installer Jamulus.
```

Comment:
The title "ASIO4ALL - Universal ASIO Driver For WDM Audio" is new, I think. Please open a pr for this for the changes branch!